index.html

```diff
@@ -1,228 +1,70 @@
-<html lang="eng">
+<!DOCTYPE html>
+<html lang="en">
 
   <head>
     <meta charset="UTF-8">  
     <meta name="viewport" content="width=device-width, initial scale=1">
+    <link rel="stylesheet" href="style.css">
     <title>Welcome to my Paradox Collection!</title>
   </head>
 
   <body>
     <header>
       <div>
-        <img 
+        <a href="index.html">
+          <img 
           src="assets/nakin.jpeg"
           width="100" height="100"
-          alt="Logo of an owl's drawing"
-        />
-        <a href="about/index.html">About</a>
-      </div>
-      <div>  
-        <input type="search"></input>
-        <button>Search</button>
+          alt="Drawing of an owl"
+          >
+        </a>
       </div>
+      <menu class="index"
+        <a href="#" title="Select One">- Uncanny Index -</a>
+        <ul>
+          <li><a href="self-fulfilling-prophesy/index.html">Self Fulfilling Prophecy/Pygmalion effect</a></li>
+          <li><a href="impostor-syndrome/index.html">Impostor Syndrome</a></li>
+          <li><a href="fermi-paradox/index.html">Fermi Paradox</a></li>
+          <li><a href="phenomenon-of-baader-meinhof/index.html">Phenomenon of Baader-Meinhof</a></li>
+          <li><a href="curse-of-knowledge/index.html">Curse of Knowledge</a></li>
+          <li><a href="stockholm-syndrome/index.html">Stockholm Syndrome</a></li>
+          <li><a href="uncanny-valley/index.html">Uncanny Valley</a></li> 
+          <li><a href="observer-expectancy-effect/index.html">Observer-expectancy effect</a></li>
+          <li><a href="base-rate-fallacy/index.html">Base rate fallacy</a></li>
+        </ul>  
+      </menu>
     </header>
+ 
+    <div class="line"></div>
 
     <main>
       <h1>Paradox Collection</h1>
       <h2>A personal list of curious psychological phenomena, paradoxes and cognitive biases</h2>
     </main>
```

Comment:
I would say `<main>` should represent the main content of each page and thus wrap everything that's not header and footer (in this implementation, the "line" divs are part of the header and footer so keep them out of `<main>`)